examples/tutorials/core_api_pytorch_mnist/model_def_checkpoints.py

```diff
@@ -97,11 +100,23 @@ def test(args, model, device, test_loader, epoch, core_context, steps_completed)
     )
 
 
-# NEW: Define load_state function for restarting model training from existing checkpoint.
-def load_state(checkpoint_directory):
+# NEW: Define load_state function for restarting model training from existing checkpoint. Returns (.pt, int).
+# Also update load_state header to take trial info object as an argument.
+def load_state(checkpoint_directory, info):
```

Comment:
accept a `trial_id: int` here instead of an `info: ClusterInfo`.  No need to pass a huge object just for one integer it contains.